### PR TITLE
⭐️ experimental update sub-command

### PR DESCRIPTION
**Pull request by chris-rock**
This PR adds the experimental update command to cnspec

```
cnspec update
```

It download the install script and runs it locally.

### CODE REVIEW

File: apps/cnspec/cmd/update.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+	_, err = file.Write(scriptData)
+	if err != nil {
+		return errors.Wrap(err, "could not write downloaded script")
+	}
+
+	log.Info().Str("script", file.Name()).Msg("script downloaded successfully")
+
+	log.Info().Msg("run update script")
+	cmd := exec.Command("bash", file.Name())
```
Comment (czunker) [inline apps/cnspec/cmd/update.go:83]: Not every Linux has `bash` installed. Can we changed this? Or should we check that bash is present?

**Comment by czunker** [comment]
I do not understand what this command will be used for. Why not let the package manager do the update?

**Comment by chris-rock** [comment]
> I do not understand what this command will be used for. Why not let the package manager do the update?

That only works well on Linux systems and is more difficult on windows and macOS. To ensure we just use the package manager, we rely on the install script.

Essentially this command makes it easier for users, since the do not need to manually download the script and run the script. It came up quite a lot as feedback from Windows users.

File: apps/cnspec/cmd/update.go (hunk 1 above)
Comment (chris-rock) [reply apps/cnspec/cmd/update.go:83]: Right now the script requires bash, therefore we should check for its existence. Great point.

**Comment by chris-rock** [comment]
@czunker PR is updated and checks for bash now

**Review by czunker — APPROVED** [review]
Worked with my tests on a Linux box.

Thanks @chris-rock

**Pull request merged**